ReplayBrowser/Startup.cs

```diff
@@ -86,6 +86,8 @@ public void ConfigureServices(IServiceCollection services)
         services.AddSingleton<AnalyticsService>();
         services.AddSingleton<NoticeHelper>();
         services.AddSingleton<GitHubApiHelper>();
+        
+        services.AddMemoryCache();
```

Comment:
We already have a memory cache, this is not needed.

Reply:
ogh sorry

Reply:
It's fine, if a cache is already added, this method will do nothign AFAIK.